packages/frontend/src/main.tsx

```diff
@@ -4,10 +4,14 @@ import React from 'react'
 import ReactDOM from 'react-dom/client'
 
 import { App } from './App'
+import { getConfig } from './config'
+
+// eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+const config = getConfig(import.meta.env.VITE_DEPLOYMENT_ENV ?? 'staging')
```

Comment:
I'm not sure about placing config dependency here. Maybe we should migrate this down to the app file? Outside the component